doc/user/content/ingest-data/network-security/static-ips.md

```diff
@@ -15,32 +15,46 @@ When connecting Materialize to services in your private networks (e.g., Kafka
 clusters or PostgreSQL databases), you must configure any firewalls to allow
 connections from these IP addresses.
 
-## Details
+## Static egress IPs
 
 Your Materialize region is associated with a static set of egress IP addresses.
 Most regions have four egress IP addresses, but this is not guaranteed. All
 connections to the public internet initiated by a source or sink in your region
 will originate from one of these egress IP addresses.
 
-We do not allocate unique IP addresses for each Materialize region. Multiple
-regions may share the same egress IP addresses.
+We allocate unique IP addresses for each Materialize region; however, multiple
+regions may share the same egress IP address CIDR.
 
 When connecting Materialize to services in your private networks (e.g., Kafka
-clusters or PostgreSQL databases), you must configure any firewalls to allow
-connections from your region's egress IP addresses. You must allow connections
-from all egress IP addresses associated with your region. Connections may
-originate from any one of the egress IP addresses.
+clusters or PostgreSQL databases),
+-  you must configure any firewalls to allow connections from your region's
+egress IP addresses.
+- You must allow connections from all egress IP addresses associated with your
```

Comment:
these bullet points feel like they can be combined... feels like the original phrasing was fine, except missing the part where we specify CIDRs now